src/render/quantization.cpp

```diff
@@ -176,8 +201,11 @@ Image* convert_pixel_format(
 
             if (a == 0)
               *dst_it = new_mask_color;
-            else if (rgbmap)
-              *dst_it = rgbmap->mapColor(r, g, b, a);
+            if (octreeMap)
```

Comment:
I think it requires an `else` (because the first case `(a == 0)` was already handled:
```suggestion
            else if (octreeMap)
```